sg20/d1389/d1389.bs

```diff
@@ -323,27 +323,46 @@ TODO (why?)
 
 TODO (why?)
 
-### [tools.debugger] Introduce a debugger early
+### [tools.testing] Introduce a testing framework
+
+Examples: [Catch2](https://github.com/catchorg/Catch2), [Google Test](https://github.com/google/googletest)
 
 TODO (why?)
 
+### [tools.debugger] Introduce a debugger early
+
+The ability to step through running code and examine execution state will enable students to troubleshoot issues and correct assumptions they have made about the behavior of language and library features.
+
 ### [tools.package.management] Use a package manager
 
-TODO (why?)
+Examples: [Vcpkg](https://github.com/Microsoft/vcpkg), [Conan](https://conan.io/)
+
+Downloading, installing, and building against dependencies in C++ can be a challenge, especially for beginners. Package managers help alleviate this by providing tested packages along with automatic installation scripts.
```

Comment:
+1

You don't need to do this, but I think a section discussing the pros and cons of both (or linking to a resource that does so) would be beneficial.